### PR TITLE
feat(keybindings): keybindings for Graph, All Pages, and Settings

### PR DESCRIPTION
**Pull request by sawhney17**
Add keyboard shortcuts for Graph view, Pages view and Settings view

### CODE REVIEW

**Comment by sawhney17** [comment]
I've added some new keybindings. I followed what made logical sense to me. 

For mac
Option-A: All Pages view
Option-G: Graph View
CMD-COMMA: Settings
For Windows: 
alt-A: All Pages view
alt-G: Graph View
ctrl-COMMA: Settings

First two are in line with what we have done with the daily note shortcut
Cmd-Comma is pretty much the standard across every single app.

**Comment by Limezy** [comment]
⌘, is very common for the settings on Mac, agreed
⌘G for **G**raph view seems pretty legit
However, I would expect ⌘A to select all blocks in the page, no ?

**Comment by sawhney17** [comment]
> ⌘G for **G**raph view seems pretty legit
> However, I would expect ⌘A to select all blocks in the page, no ?

Both of these are **option/alt** not **command/ctrl**. It's similar to the daily notes which is option/alt d. 

Command G is reserved for the querying function or something. Looks like this 
Screen Shot 2021-05-18 at 6 29 26 PM

**Comment by agentydragon** [comment]
Maybe you could also add the shortcuts into athens/src/cljs/athens/athens_datoms.cljs so people can discover them?

File: src/cljs/athens/listeners.cljs [hunk 1]
```diff
@@ -100,12 +100,17 @@
...
+                                       KeyCodes.COMMA (router/navigate :settings)
+
                                        KeyCodes.T (util/toggle-10x)
                                        nil)
       alt (condp = key-code
             KeyCodes.LEFT (when (nil? editing-uid) (.back js/window.history))
             KeyCodes.RIGHT (when (nil? editing-uid) (.forward js/window.history))
             KeyCodes.D (router/nav-daily-notes)
+            KeyCodes.G (router/navigate :graph)
+            KeyCodes.A (router/navigate :pages)
```
Comment (arkRedM) [inline src/cljs/athens/listeners.cljs:113]: Please do add shortcuts to initial datoms for discoverability until a help popover comes along

**Review by arkRedM — CHANGES_REQUESTED** [review]
Very minor suggestions

**Comment by sawhney17** [comment]
@arkRedM I'll get right to adding all of this

**Comment by Limezy** [comment]
> Both of these are **option/alt** not **command/ctrl**. It's similar to the daily notes which is option/alt d.
Yep sorry I was too fast !

**Review by tangjeff0 — APPROVED** [review]
Way to go @sawhney17 ! Getting into the code 😎

**Pull request merged**